fsh/ig-data/input/pages/8_qualityMeasures.md

```diff
@@ -1,3 +1,13 @@
+<!-- 8_qualityMeasures.md {% comment %}
+*****************************************************************************************
+*                            WARNING: DO NOT EDIT THIS FILE                             *
+*                                                                                       *
+* This file is generated by SUSHI. Any edits you make to this file will be overwritten. *
+*                                                                                       *
+* To change the contents of this file, edit the original source file at:                *
+* ig-data\input\pagecontent\8_qualityMeasures.md                                        *
+*****************************************************************************************
+{% endcomment %} -->
```

Comment:
Why Do we have generated files in the repo? Can't we generate them in Travis?

Reply:
These files should definitly not be in the repo. I'll investigate how they got there....

Reply:
Was copy-paste error. (Overwrote source files with files copied by SUSHI when restoring an earlier version).